### PR TITLE
fix: Fix exception logging after analysis #546

### PR DESCRIPTION
**Pull request by ap891843**
There was a bug with the initial implementation. The error logs of doc analysis were not logged.

Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

**Comment by temanbrcom** [comment]
@asatklichov 
@grianbrcom 
@Nurkambay

File: server/src/main/java/com/broadcom/lsp/cobol/service/CobolTextDocumentService.java [hunk 1]
```diff
@@ -251,21 +251,23 @@ private void clearAnalysedFutureObject(String uri) {
...
+                            ? CopybookProcessingMode.ENABLED_VERBOSE
+                            : CopybookProcessingMode.ENABLED);
                 AnalysisResult result = engine.analyze(uri, text, copybookProcessingMode);
                 ofNullable(docs.get(uri)).ifPresent(doc -> doc.setAnalysisResult(result));
                 publishResult(uri, result, copybookProcessingMode);
                 outlineMap.get(uri).complete(result.getOutlineTree());
               } catch (Exception e) {
-                reportExceptionIfThrown(createDescriptiveErrorMessage("analysis", uri));
+                reportExceptionIfThrown(createDescriptiveErrorMessage("analysis", uri))
+                    .accept(null, e);
```
Comment (grianbrcom) [inline server/src/main/java/com/broadcom/lsp/cobol/service/CobolTextDocumentService.java:270]: Oh no =) It has become quite complicated. Could you please change it to:
```
LOG.error(createDescriptiveErrorMessage("analysis", uri), e);
```

**Review by grianbrcom — CHANGES_REQUESTED** [review]
Please, simplify a couple of lines.

File: server/src/main/java/com/broadcom/lsp/cobol/service/CobolTextDocumentService.java (hunk 1 above)
Comment (ap891843) [reply server/src/main/java/com/broadcom/lsp/cobol/service/CobolTextDocumentService.java]: Done.

**Review by temanbrcom — APPROVED** [review]

**Review by grianbrcom — APPROVED** [review]
:whale:

**Pull request merged**